src/plugins/vis_type_table/public/components/table_vis_component.tsx

```diff
@@ -107,6 +107,13 @@ export const TableVisComponent = ({
 
   const ariaLabel = title || visConfig.title || 'tableVis';
 
+  const footerCellValue = visConfig.showTotal
+    ? ({ columnId }: { columnId: any }) => {
+        const colIndex = columns.findIndex((col) => col.id === columnId);
+        return columns[colIndex]?.formattedTotal || null;
```

Comment:
Couldn't we do something slightly simpler like:

```typescript
return columns.find((col) => col.id === columnId)?.formattedTotal || null;
```

Reply:
yeah nice. will change. thx.